staging/src/k8s.io/genericapiserver/pkg/authentication/user/doc.go

```diff
@@ -16,4 +16,4 @@ limitations under the License.
 
 // Package user contains utilities for dealing with simple user exchange in the auth
 // packages. The user.Info interface defines an interface for exchanging that info.
-package user // import "k8s.io/kubernetes/pkg/auth/user"
+package user // import "k8s.io/genericapiserver/pkg/authentication/user"
```

Comment:
@lavalamp The pull looks massive, but this is the meat right here.  Moving three files.